uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Upgrader/Step/1022.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+trait CRM_HRLeaveAndAbsences_Upgrader_Step_1022 {
+  
+  /**
+   * Default existing work pattern to As per contract (new default)
+   *
+   * @return bool
+   */
+  public function upgrade_1022()
+  {
+    civicrm_api3('ContactWorkPattern', 'get', [
+      'return' => ['id'],
+      'api.ContactWorkPattern.update' => [
```

Comment:
I think `update` is deprecated, in the API explorer it has a ~strikethrough~ effect. Can you test if this works with `create` and change it if it does?